packages/core/custom-elements-lsp/package.json

```diff
@@ -1,91 +1,92 @@
-  "version": "2.0.1",
-  "license": "MIT",
-  "main": "./out", "bin": {
-    "custom-elements-analyze": "./parser/analyze.js"
-  },
-  "description": "TypeScript plugin to enable Intellisense in custom element tagged templates",
-  "keywords": [
-    "analyzer",
-    "autocompletion",
-    "definition",
-    "diagnostics",
-    "fast",
-    "intellisense",
-    "javascript",
-    "jump",
-    "lsp",
-    "nvim",
-    "quickinfo",
-    "typescript",
-    "vscode",
-    "warnings"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/genesiscommunitysuccess/custom-elements-lsp"
-  },
-  "publishConfig": {
-    "access": "public"
-  },
-  "homepage": "https://github.com/genesiscommunitysuccess/custom-elements-lsp",
-  "bugs": {
-    "url": "https://github.com/genesiscommunitysuccess/custom-elements-lsp/issues"
-  },
-  "scripts": {
-    "bootstrap:module": "npm run build",
-    "build": "npx tsc -b ./src/tsconfig.json",
-    "build:watch": "npx tsc --watch",
-    "clean:dist": "rimraf ./out",
-    "copy-manifest": "cp ./example/ce.json ./src/jest/ce-test.json",
-    "release": "semantic-release",
-    "test:unit": "jest",
-    "test:unit:coverage": "jest --coverage",
-    "test:unit:debug": "node --inspect-brk node_modules/jest/bin/jest.js",
-    "test:unit:verbose": "cross-env TEST_LOG=1 jest",
-    "test:unit:watch": "jest --watchAll"
-  },
-  "dependencies": {
-    "@custom-elements-manifest/analyzer": "^0.8.0",
-    "chokidar": "^3.5.3",
-    "debounce": "^1.2.1",
-    "get-tsconfig": "^4.7.0",
-    "globby": "^13.1.4",
-    "minimist-lite": "^2.2.1",
-    "node-html-parser": "^6.1.5",
-    "resolve": "^1.22.4",
-    "resolve-pkg": "^2.0.0",
-    "typescript-template-language-service-decorator": "^2.3.2"
-  },
-  "devDependencies": {
-    "@commitlint/cli": "^17.6.7",
-    "@commitlint/config-conventional": "^17.6.7",
-    "@commitlint/format": "^17.4.4",
-    "@semantic-release/changelog": "^6.0.2",
-    "@semantic-release/git": "^10.0.1",
-    "@semantic-release/release-notes-generator": "^11.0.4",
-    "@types/debounce": "^1.2.1",
-    "@types/jest": "^29.5.1",
-    "@types/resolve": "^1.20.2",
-    "@typescript-eslint/eslint-plugin": "^5.2.0",
-    "@typescript-eslint/parser": "^5.2.0",
-    "conventional-changelog-conventionalcommits": "^5.0.0",
-    "cross-env": "^7.0.3",
-    "custom-elements-manifest": "^2.0.0",
-    "eslint": "8.22.0",
-    "eslint-config-google": "^0.14.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-config-ts-important-stuff": "^1.1.0",
-    "eslint-import-resolver-typescript": "^3.5.3",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-unused-imports": "^2.0.0",
-    "husky": "^8.0.3",
-    "jest": "^29.5.0",
-    "lint-staged": "^13.2.2",
-    "semantic-release": "21.0.7",
-    "ts-jest": "^29.1.0",
-    "typescript": "^4.4.3"
-  }
+	"name": "@genesiscommunitysuccess/custom-elements-lsp",
```

Comment:
nit: 

there's something happening with the formatting here 👀

Reply:
Fixed in latest push